build.sbt

```diff
@@ -183,8 +183,27 @@ lazy val tla_bmcmt = (project in file("tla-bmcmt"))
       libraryDependencies += Deps.scalaCollectionContrib,
   )
 
+lazy val shai = (project in file("shai"))
```

Comment:
Correction location in in file (should be prior to tool, since tool depends on it) and name of project.

Reply:
I find it oddly gratifying that this project is (partially) called like the Fremen word for sandworm 😅